care/facility/api/viewsets/patient.py

```diff
@@ -648,3 +659,23 @@ def perform_create(self, serializer):
             patient=patient,
             created_by=self.request.user,
         )
+
+    def perform_update(self, serializer):
```

Comment:
Reimplement this feature in the model save method.

Reply:
Done.